control/config_proxy.go

```diff
@@ -0,0 +1,104 @@
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2016 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package control
+
+import (
+	"encoding/json"
+
+	"golang.org/x/net/context"
+
+	"github.com/intelsdi-x/snap/control/rpc"
+	"github.com/intelsdi-x/snap/core"
+	"github.com/intelsdi-x/snap/core/cdata"
+	"github.com/intelsdi-x/snap/internal/common"
+)
+
+type configProxy struct {
+	cfg *Config
+}
+
+func (cfg *configProxy) GetPluginConfigDataNode(ctx context.Context, r *rpc.ConfigDataNodeRequest) (*rpc.ConfigDataNode, error) {
+	reply := &rpc.ConfigDataNode{}
+	Type := int(r.PluginType)
+	Name := r.Name
+	Version := int(r.Version)
+	result_node := cfg.cfg.GetPluginConfigDataNode(core.PluginType(Type), Name, Version)
+	reply.Node, _ = nodeToJSON(&result_node)
```

Comment:
I don't have a problem either way, just as long as we're consistent for everything.